modules/integration_azure-container-apps/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,16 @@
+module: Azure Container Apps
+name: heartbeat
+filtering: "filter('resource_type', 'Microsoft.App/containerApps') and filter('aggregation_type', 'minimum')"
+
+transformation: true
+aggregation: ".mean(by=['azure_resource_id'])"
```

Comment:
```suggestion
aggregation: ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region', 'azure_tag_env'])"
```

Reply:
No, `azure_tag_env` is not generic. It should be in templates, not module.